### PR TITLE
Add blacklist script

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Script to add addresses to blacklist](https://trello.com/c/BKL2BtF0/212-script-to-add-addresses-to-blacklist)

File: bin/lamassu-blacklist [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  input: process.stdin,
+  output: process.stdout
+})
+
+const sql = `select regexp_replace(crypto_code, '$', ' ') as code,regexp_replace(address, '^', ' ') as address from blacklist`
+const insertSql = `insert into blacklist(crypto_code, address) values ($1, $2)`
+
+db.query(sql)
+  .then(many => {
+    console.log('')
```
Comment (joshmh) [inline bin/lamassu-blacklist]: I think you can just use: ``console.log()``.

File: bin/lamassu-blacklist [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    console.log('Here is your current list of prohibited addresses:')
```
Comment (joshmh) [inline bin/lamassu-blacklist]:
```suggestion
    console.log('Here is your current list of prohibited addresses:\n')
```

Then you don't need all the empty ``console.log``s.

File: bin/lamassu-blacklist [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    console.log('')
+    console.log(many.map(it => `${it.code} | ${it.address}`).join(os.EOL))
+    console.log('')
+    console.log('What address would you like to add to the list?')
+    console.log('')
+    console.log('\x1b[2m%s\x1b[0m', 'Example:', 'bc1q8pu2zlfxg8jf4cyuf844rl3uxmmw8sj9yaa393')
```
Comment (joshmh) [inline bin/lamassu-blacklist]: What do these escape sequences do? How about using https://github.com/chalk/chalk instead?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: bin/lamassu-blacklist (hunk 3 above)
Comment (RafaelTaranto) [reply bin/lamassu-blacklist]: I though about adding chalk but I don't think it is worth adding a dependency at this point. 

The escape sequences dim the first string so we give a different look to `Example:`.

I can either add chalk or name the escape sequences, up to you.

File: bin/lamassu-blacklist (hunk 2 above)
Comment (RafaelTaranto) [reply bin/lamassu-blacklist]: I've used os.EOL.

**Pull request merged**